plugins/source/pagerduty/resources/plugin/all_tables.go

```diff
@@ -37,4 +38,12 @@ func AllTables() []*schema.Table {
 		users.Users(),
 		vendors.Vendors(),
 	}
+	if err := transformers.TransformTables(t); err != nil {
+		panic(err)
+	}
+	for _, table := range t {
+		schema.AddCqIDs(table)
+		titleTransformer(table)
```

Comment:
This doesn't apply the transformer to child tables of `table`, passing it to `transformers.Apply` would do it.

Reply:
Good point, thanks, updated 👍